src/cdk/collections/selection.ts

```diff
@@ -34,7 +34,13 @@ export class SelectionModel<T> {
   }
 
   /** Event emitted when the value has changed. */
-  onChange: Subject<SelectionChange<T>> | null = this._emitChanges ? new Subject() : null;
+  changed: Subject<SelectionChange<T>> | null = this._emitChanges ? new Subject() : null;
```

Comment:
Didn't we land on `change` present-tense?

Reply:
We landed on `myEventThingChange` but the rest of them ended up past tense.

Reply:
Which things are `changed`?

Reply:
A few I know of `selectedChanged`, `_scrollDistanceChanged` and `_selectedIndexChanged`. Let me know what you want to do.